cassiopeia/solver/DistanceSolver.py

```diff
@@ -75,6 +75,39 @@ def __init__(
         self.dissimilarity_function = dissimilarity_function
         self.add_root = add_root
 
+    def get_dissimilarity_map(
+        self, 
+        cassiopeia_tree: CassiopeiaTree,
+        layer: Optional[str] = None
+    ) -> pd.DataFrame: 
+        """Obtains or generates a matrix that is updated throughout the solve.
+
+        The highest-level method to obtain a dissimilarity map, which will be 
+        the matrix primarily used throughout the solve method. This matrix 
+        contains information for decisions related to clade formation, and 
+        will be updated at every iteration within the solve method. This method 
```

Comment:
Better to say "This matrix contains the pairwise dissimilarity between samples which is used for identifying sample pairs to merge, and will be updated at every iteration within the solve method.